swarm.go

```diff
@@ -35,6 +35,9 @@ type SwarmConfig struct {
 	// When enabled, node will provide a limited relay service to other peers.
 	RelayService RelayService
 
+	// EnableHolePunching enables the hole punching service.
+	EnableHolePunching Flag
```

Comment:
I think this should be skipped from JSON unless someone really, really wants to change the default.

```suggestion
	EnableHolePunching Flag `json:",omitempty"`
```